docs/conf.py

```diff
@@ -57,7 +57,7 @@
 html_theme_options = {
     "description": "A client library to communicate with StArMap",
     "extra_nav_links": {
-        "Source": "https://gitlab.cee.redhat.com/stratosphere/starmap-client",
+        "Source": "https://github.com/release-engineering/starmap-client",
```

Comment:
@jajreidy If you have some time, it's a good idea to change these URLs for [cloudpub](https://github.com/release-engineering/cloudpub) and [pubtools](https://github.com/release-engineering/pubtools-marketplacesvm) as well as their documentations are sending the `Source` back to the internal gitlab 😓